src/main/java/com/nitnelave/CreeperHeal/PluginHandler.java

```diff
@@ -18,21 +18,22 @@
 public class PluginHandler
 {
 
-    private static MobArenaHandler maHandler = null;
-    private static LWC lwc = null;
-
-    private static final boolean spout;
+    private static final MobArenaHandler maHandler;
+    private static final LWC lwc;
 
     static
     {
         Plugin lwcp = detectPlugin("LWC");
         if (lwcp != null)
             lwc = ((LWCPlugin) lwcp).getLWC();
+        else
+            lwc = null;
```

Comment:
This is not needed, static variables are initialized at the beginning of the execution, with null.

Reply:
Fair. My compiler complains about the initialization because it's done in and outside of the static block, but that's easy to remove.